src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) throws InterruptedException {
+        Machine[] machines = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
+
+        for (Machine machine : machines) {
```

Comment:
Remove empty lines.
"sleep" is not needed here

Reply:
done